platforms/core-runtime/classloaders/src/main/java/org/gradle/internal/classpath/DefaultClassPath.java

```diff
@@ -217,7 +220,7 @@ public ClassPath build() {
         }
     }
 
-    protected static final class ImmutableUniqueList<T> extends AbstractList<T> implements Serializable {
+    public static final class ImmutableUniqueList<T> extends AbstractList<T> implements Serializable {
```

Comment:
Why do we need this?

Reply:
It was a warning in the file, this class is used as return value of a public method.